tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -40,20 +42,22 @@ bool IsRetriable(Status status) {
 Status CallWithRetries(const std::function<Status()>& f) {
```

Comment:
I don't think CallWithRetries is called from the unittest directly, so how would you make sure that initial_delay_seconds is set to 0 for tests?